scripts/mitochondria_m2_wdl/AlignAndCall.wdl

```diff
@@ -523,6 +527,7 @@ task Filter {
     Boolean compress
     Float? vaf_cutoff
     String sample_name
```

Comment:
Since you're switching to `base_name` can you delete `sample_name` from the whole pipeline? Does it get used anywhere? I didn't see it in this task.